README.md

```diff
@@ -3,7 +3,7 @@
 
 # Solidus Starter Frontend
 [![CircleCI](https://circleci.com/gh/solidusio/solidus_starter_frontend.svg?style=shield)](https://circleci.com/gh/solidusio/solidus_starter_frontend)
-[![codecov](https://codecov.io/gh/solidusio/solidus_starter_frontend/branch/master/graph/badge.svg?token=54gge25dNh)](https://codecov.io/gh/solidusio/solidus_starter_frontend)
+[![codecov](https://codecov.io/gh/solidusio/solidus_starter_frontend/branch/main/graph/badge.svg?token=54gge25dNh)](https://codecov.io/gh/solidusio/solidus_starter_frontend)
```

Comment:
This change is needed because this repo already uses main as default branch, right?

Reply:
Exactly :slightly_smiling_face: